app/scripts/modules/core/delivery/service/execution.service.js

```diff
@@ -290,8 +291,19 @@ module.exports = angular.module('spinnaker.core.delivery.executions.service', [
             }
           }
         }
-        current.stringVal = updated.stringVal;
       });
+      current.stringVal = updated.stringVal;
```

Comment:
This isn't a big deal (and probably has no performance impact), but the assignment was happening inside the iteration over the stages, so just moving out to a more sensible place.